great_expectations/expectations/metrics/column_aggregate_metrics/column_values_length_min.py

```diff
@@ -23,9 +23,9 @@ def _pandas(cls, column: pd.Series, **kwargs: dict) -> int:
     @column_aggregate_partial(
         engine=SqlAlchemyExecutionEngine, filter_column_isnull=True
     )
-    def _sqlalchemy(cls, column, **kwargs: dict):  # type: ignore[no-untyped-def]
+    def _sqlalchemy(cls, column, **kwargs: dict):
         return sa.func.min(sa.func.length(column))
 
     @column_aggregate_partial(engine=SparkDFExecutionEngine, filter_column_isnull=True)
-    def _spark(cls, column, **kwargs: dict):  # type: ignore[no-untyped-def]
+    def _spark(cls, column, **kwargs: dict):
```

Comment:
Can we give `column` a type?
Same for question for`column_values_length_max.py` (and others)